### PR TITLE
MailUtility

### PR DESCRIPTION
**Pull request by KalpeshChaudhari**
#17 Added MailUtility.java for sending email.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,6 +32,8 @@ dependencies {
 	compile group: 'com.fasterxml.jackson.dataformat', name: 'jackson-dataformat-yaml', version: jacksonDataformatVersion
 	compile group: 'org.yaml', name: 'snakeyaml', version: snakeYamlVersion
 	compile group: 'io.github.bonigarcia', name: 'webdrivermanager', version: webDriverManagerVersion
+	compile group: 'javax.mail', name: 'mail', version: '1.4.1'
```
Comment (amitmindstix) [inline build.gradle]: Version cannot be hardcoded - look at other entries

File: src/test/resources/emailconfig.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#Gmail Login Credentials
+mailAddress= YOUR_EMAIL_ADDRESS
```
Comment (amitmindstix) [inline src/test/resources/emailconfig.properties]: How does it get injected here? Need some notes on need to update this file.

**Review by amitmindstix — CHANGES_REQUESTED** [review]
See comments

File: src/test/resources/emailconfig.properties [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#Gmail Login Credentials
```
Comment (amitmindstix) [inline src/test/resources/emailconfig.properties]: Does it really need to be Gmail login? Why not yahoo or outlook or some other email provider?

File: src/test/resources/emailconfig.properties (hunk 2 above)
Comment (KalpeshChaudhari) [reply src/test/resources/emailconfig.properties]: I think we need to add this in readme file?

Comment (amitmindstix) [reply src/test/resources/emailconfig.properties]: Lets add at least at the class level in javadoc mentioning the need to update the config file with parameters that are mandatory

Comment (amitmindstix) [reply src/test/resources/emailconfig.properties]: We may even catch the condition where the email, pwd are still default - then throw an exception RuntimeException("emailconfig not initialized with credentials!")

File: src/main/java/com/mindstix/cb/utils/PropertiesUtility.java [hunk 4]
```diff
@@ -105,6 +123,11 @@ private static void loadPoperties() {
 				testData.load(dataInput);
 				dataInput.close();
 				LOGGER.info("Loading Test Data properties");
+				dataInput = new FileInputStream("src/test/resources/emailconfig.properties");
+				emailConfigs.load(dataInput);
+				dataInput.close();
+				LOGGER.info("Loading Email config properties");
```
Comment (amitmindstix) [inline src/main/java/com/mindstix/cb/utils/PropertiesUtility.java:129]: Loading? or Loaded?

**Comment by amitmindstix** [comment]
Could you create a PR on v0.1-alpha instead of master?

**Pull request closed without merging**